openreview/journal/group.py

```diff
@@ -80,56 +127,33 @@ def set_groups(self, support_role, editors):
-
-        header['instructions'] = f'''
-        <p>
-            <strong>Editors-in-chief:</strong></br>
-            {editors}
-            <strong>Managing Editors:</strong></br>
-            <a href=\"https://openreview.net/profile?id={support_role}\">{openreview.tools.pretty_id(support_role)}</a>
-        </p>
-        <p>
-            {self.journal.full_name} ({venue_id}) is a venue for dissemination of machine learning research that is intended to complement JMLR while supporting the unmet needs of a growing ML community.
-        </p>
-        <ul>
-            <li>
-                <p>{venue_id} emphasizes technical correctness over subjective significance, in order to ensure we facilitate scientific discourses on topics that are deemed less significant by contemporaries but may be so in the future.</p>
-            </li>
-            <li>
-                <p>{venue_id} caters to the shorter format manuscripts that are usually submitted to conferences, providing fast turnarounds and double blind reviewing. </p>
-            </li>
-            <li>
-                <p>{venue_id} employs a rolling submission process, shortened review period, flexible timelines, and variable manuscript length, to enable deep and sustained interactions among authors, reviewers, editors and readers.</p>
-            </li>
-            <li>
-                <p>{venue_id} does not accept submissions that have any overlap with previously published work.</p>
-            </li>
-        </ul>
-        <p>
-            For more information on {venue_id}, visit
-            <a href="http://{self.journal.contact_info}" target="_blank" rel="nofollow">{self.journal.contact_info}</a>.
-        </p>
-        '''
+            editors+=f'[{name}](https://openreview.net/profile?id={m})  \n'
+
+        instructions = f'''
+**Editors-in-chief:**  
+{editors}
+
+**Managing Editors:**  
+[{openreview.tools.pretty_id(support_role)}](https://openreview.net/profile?id={support_role})
+
+{self.journal.full_name} ({venue_id}) is a venue for dissemination of machine learning research that is intended to complement JMLR while supporting the unmet needs of a growing ML community.  
+
+- {venue_id} emphasizes technical correctness over subjective significance, in order to ensure we facilitate scientific discourses on topics that are deemed less significant by contemporaries but may be so in the future.
+- {venue_id} caters to the shorter format manuscripts that are usually submitted to conferences, providing fast turnarounds and double blind reviewing.
+- {venue_id} employs a rolling submission process, shortened review period, flexible timelines, and variable manuscript length, to enable deep and sustained interactions among authors, reviewers, editors and readers.
+- {venue_id} does not accept submissions that have any overlap with previously published work.
+
+For more information on {venue_id}, visit [{self.journal.contact_info}](http://{self.journal.contact_info})
```

Comment:
i think this contact_info is different from existing webfield
it's an email tmlr@jmlr.org
in current webfield it's a url jmlr.org/tmlr

Reply:
Fixed. Thanks for the catch.